python/ray/worker.py

```diff
@@ -129,6 +130,23 @@ def __str__(self):
         return "\n".join(out)
 
 
+TaskContext = namedtuple('TaskContext', [
+    'current_task_id',
+    'task_index',
+    'put_index',
```

Comment:
This should probably also include the current task's driver ID since this changes for non-actors.

Reply:
`task_driver_id` isn't  thread-specific. At any moment, all threads should have the same driver id. So it shouldn't be here. Does that make sense?